sql/src/main/java/org/apache/druid/sql/SqlLifecycle.java

```diff
@@ -248,46 +256,44 @@ private ValidationResult validate(AuthenticationResult authenticationResult)
     }
   }
 
-  private Access doAuthorize(final Access authorizationResult)
+  private void doAuthorize(Function<Set<ResourceAction>, Access> authorizer)
   {
+    Access authorizationResult = planner.authorize(
+        authorizer,
+        authConfig.authorizeQueryContextParams());
```

Comment:
super nit: formatting, the last `)` should be on the following line, this is one of those edge cases the style bots don't catch for whatever reason (i seem to recall it being some limitation in .. something.. somewhere).